ros2_batch_job/__main__.py

```diff
@@ -517,14 +527,6 @@ def run(args, build_function, blacklisted_package_names=None):
         # Install pip dependencies
         pip_packages = list(pip_dependencies)
 
-        def need_package_from_pipy(pkg_name):
-            try:
-                importlib.import_module(pkg_name)
-            except ModuleNotFoundError:
-                return True
-
-            return False
-
```

Comment:
This function isn't removed it's just relocated so it can be used earlier.

https://github.com/ros2/ci/pull/643/files#diff-54f7a5a1191410c54331d5e337e5760c7de5e219f01dfc732b9380dbf71b46b7R508-R515